pallets/aggregate/src/migrations/v2.rs

```diff
@@ -0,0 +1,214 @@
+            max_aggregation_size: 32,
+            should_publish: BoundedBTreeMap::new(),
+            publish_queue_size: 5,
+            ticket: ticket_owner.map(|who| MockConsideration {
+                who,
+                count: 10,
+                size: 1000,
+            }),
+            aggregate_rules: v1::AggregateSecurityRules::Untrusted,
+            delivery: v1::DeliveryParams {
+                owner: 123_u64,
+                data: v1::Delivery {
+                    destination: Destination::None,
+                    fee: 100,
+                },
+            },
+        }
+    }
+
+    #[test]
+    fn successful_migration() {
+        test().execute_with(|| {
+            // CLEAN THE TEST STORAGE
+            v1::Domains::<Test>::drain().count();
+
+            v1::Domains::<Test>::insert(
+                23,
+                create_old_domain(23, v1::User::from(123), v1::DomainState::Ready, Some(321)),
+            );
+            v1::Domains::<Test>::insert(
+                42,
+                create_old_domain(42, v1::User::from(321), v1::DomainState::Hold, Some(123)),
+            );
+            // This one is create by manager: could not be migrated
+            v1::Domains::<Test>::insert(
+                1,
+                create_old_domain(1, v1::User::Manager, v1::DomainState::Removable, None),
+            );
+            v1::Domains::<Test>::insert(
+                2,
+                create_old_domain(2, v1::User::from(42), v1::DomainState::Removable, Some(33)),
+            );
+
+            #[cfg(feature = "try-runtime")]
+            let expected_encoded = InnerMigrateV1ToV2::<Test>::pre_upgrade()
+                .map_err(|e| format!("pre_upgrade failed: {:?}", e))
+                .unwrap();
+
+            // Perform runtime upgrade
+            let weight = InnerMigrateV1ToV2::<Test>::on_runtime_upgrade();
+
+            #[cfg(feature = "try-runtime")]
+            assert_ok!(InnerMigrateV1ToV2::<Test>::post_upgrade(expected_encoded));
+
+            // Check that `Domains` contains the expected number
+            assert_eq!(crate::Domains::<Test>::iter().count(), 3);
+
+            assert!(
+                crate::Domains::<Test>::get(1).is_none(),
+                "Domain 1 should not exist because created by the manager"
+            );
+
+            let domain_data = |id| {
+                let crate::Domain::<Test>(crate::data::DomainEntry {
+                    owner,
+                    state,
+                    ticket,
+                    ..
+                }) = crate::Domains::<Test>::take(id).unwrap();
+                (owner, state, ticket.unwrap().who)
+            };
+            use crate::data::{DomainState, User};
+            assert_eq!(domain_data(23), (User::from(123), DomainState::Ready, 321));
+            assert_eq!(domain_data(42), (User::from(321), DomainState::Hold, 123));
+            assert_eq!(domain_data(2), (User::from(42), DomainState::Removable, 33));
```

Comment:
Please, can you also che that all delivery fee are correctly set to 0: it's the main business logic in migrate function